config/deploy.reference.hocon

```diff
@@ -6,6 +6,11 @@
     // Configuration for lint command
     "lint": {
 
+      // Skip linting specified schemas in iglu:<URI> format
```

Comment:
`reference.conf` is being used as a default, does pre-filling the list w/ `iglu:com.acme/click/jsonschema/1-0-1` make sense in that context?